site/resources/js/enhancement.js

```diff
@@ -469,6 +469,7 @@ function toggleInputGroupFocus(event) {
   var classList = event.target.classList;
   var closestSelector;
   // target within InputGroup
+  if (!classList) return;
```

Comment:
Strange, why would this ever happen?

Reply:
FWIW, it's happening in production on any page.